tools/c7n_mailer/README.md

```diff
@@ -164,6 +169,7 @@ policies:
     actions:
       - type: notify
         template: default
+        template_format: 'html'
```

Comment:
Explicitly set the 'html' or 'text' format in the notify action section (it defaults to html).